frontend/pages/signup.tsx

```diff
@@ -12,10 +12,11 @@ export default function Signup(props: Props) {
   const [error, setError] = useState<Error | null>(null);
 
   const router = useRouter();
+  const baseUrl = process.env.NEXT_PUBLIC_BASE_URL;
```

Comment:
I think it would be interesting to add an env.txt file with the necessary sample variables. We should do it in the backend tb.

Reply:
Done